aws-rds-dbinstance/src/main/java/software/amazon/rds/dbinstance/BaseHandlerStd.java

```diff
@@ -609,7 +611,7 @@ protected boolean isDBInstanceStabilizedAfterMutate(
         return isDBInstanceAvailable(dbInstance) &&
                 isReplicationComplete(dbInstance) &&
                 isDBParameterGroupNotApplying(dbInstance) &&
-                isNoPendingChanges(dbInstance) &&
+                isNoPendingChanges(dbInstance, model) &&
```

Comment:
No need to pack this up in a single method call, given how foreign this signature looks. Let's add a separate method to check model state. It will keep these predicates signature-consistent and provide a logical equivalent of this combo-check.